internal/clients/team1/client_test.go

```diff
@@ -0,0 +1,56 @@
+package team1
+
+import (
+//	"github.com/sajari/regression"
+	"math"
+	"testing"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/gamestate"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+type testServerHandle struct {
+	clientGameState gamestate.ClientGameState
```

Comment:
i believe you can silence the currently failing test if you add `server.ServerForClient` in here.